service/history/execution/history_builder.go

```diff
@@ -314,6 +314,18 @@ func (b *HistoryBuilder) AddFailWorkflowEvent(decisionCompletedEventID int64,
 	return b.addEventToHistory(event)
 }
 
+// AddCancelWorkflowEvent adds WorkflowExecutionCanceled event to history
+func (b *HistoryBuilder) AddCancelWorkflowEvent(decisionCompletedEventID int64,
```

Comment:
Is this different to `AddWorkflowExecutionCanceledEvent` on line 491?

Reply:
looks like no!  just didn't search for the method names thoroughly enough.
removing this one.